doxygen/dox/ReferenceManual.dox

```diff
@@ -48,11 +45,14 @@ The functions provided by the HDF5 C-API are grouped into the following
 \a Core \a library: \ref H5 \ref H5A \ref H5D \ref H5E \ref H5ES \ref H5F \ref H5G \ref H5I \ref H5L
 \ref H5M \ref H5O \ref H5P \ref H5PL \ref H5R \ref H5S \ref H5T \ref H5VL \ref H5Z
 </td></tr>
-
 <tr><td colspan="3" style="border: none;">
 \a High-level \a library: \ref H5LT \ref H5IM \ref H5TB \ref H5PT \ref H5DS \ref H5DO \ref H5LR
 </td></tr>
-
+<tr>
+<td><a href="./deprecated.html">Deprecated functions</a></td>
+<td>Functions with \ref ASYNC "asynchronous variants"</td>
```

Comment:
remove text with quotes, ASYNC is defined in H5VLmodule.h line 33.

Reply:
Done.